app/src/main/java/com/lagradost/cloudstream3/ui/library/LibraryViewModel.kt

```diff
@@ -21,6 +21,8 @@ enum class ListSorting(@StringRes val stringRes: Int) {
     RatingLow(R.string.sort_rating_asc),
     UpdatedNew(R.string.sort_updated_new),
     UpdatedOld(R.string.sort_updated_old),
+    ReleaseDateNew(R.string.sort_release_date_new),
```

Comment:
This needs to be moved down after AlphabeticalA and AlphabeticalZ because the librarySortingMode is stored by its ordinal.